schema.graphql

```diff
@@ -8,11 +19,16 @@ type Token @entity {
 
 type Pair @entity {
   id: String!
+  protocol: Protocol!
   token0: Token!
   token1: Token!
   strategies: [Strategy!]! @derivedFrom(field: "pair")
   orders: [Order!]! @derivedFrom(field: "pair")
   trades: [Trade!]! @derivedFrom(field: "pair")
+  customFeeHistory: [PairTradingFee!]! @derivedFrom(field: "pair")
+  _defaultFeePPM: BigInt! # uint32
```

Comment:
I'd be extra clear and call it _defaultTradingFeePPM _customTradingFeePPM and tradingFeePPM

Reply:
done